### DISCUSSION TITLE
Mods like Test-ListPath (test URL, UNC or Local Path)

### DISCUSSION THREAD

**Discussion by KnifMelti** [Romanitho/Winget-AutoUpdate#231] [Ideas]
Feel a need to implement this too...
...should I do it with a new function/parameter or build it into `Test-ListPath/$ListPath`?

**Comment by KnifMelti** [comment]
...going for `Test-ModsPath/$ModsPath` - better to separate!

**Reply by Romanitho** [reply]
can be done inside the test-mod no ? as it i related to. don't know

**Reply by KnifMelti** [reply]
Related but not the same, just 2 parameters, fileloop etc.
Better to separate (to have more choices/different people managing the lists/mods)...

**Reply by KnifMelti** [reply]
We're using Azure DevOps with public projects to delegate development/list management

**Reply by Romanitho** [reply]
ok :)

**Reply by KnifMelti** [reply]
WinGet Land needs a lot of attention/corrections, the latest I ran into in my tests is PuTTY upgrading, succeeding but is failed when testing because the upgrade didn't delete the old ARP entry in registry (leaving 2 entries in Control Panel)...

**Reply by Romanitho** [reply]
putty need to be uninstalled... i got same issue on my PC. https://github.com/microsoft/winget-pkgs/pull/87021

**Reply by KnifMelti** [reply]
👌😍😁 as I said "`../WinGet Land needs a lot of attention/corrections/..`" and not all is as responsive as [@kevinoid](https://github.com/kevinoid)!

**Reply by Romanitho** [reply]
very true. Sometimes it's very frustrating regarding time we spend in WAU project ^^

**Reply by KnifMelti** [reply]
But.. ..it's alive (and a paradigm change)!

**Comment by KnifMelti** [comment]
#232

**Reply by KnifMelti** [reply]
...done!

**Comment by KnifMelti** [comment]
![image](https://user-images.githubusercontent.com/102996177/201525310-d53e9c61-53cc-4599-94ef-a575c7a5053b.png)
[WAU Klogg.conf.txt](https://github.com/Romanitho/Winget-AutoUpdate/files/10009230/WAU.Klogg.conf.txt) - rename to **WAU klogg.conf** and **Import** in **Klogg Highlighters**... (winget-install.ps1 -AppIDs **variar.klogg**)
